tests/scripts/thread-cert/Cert_5_3_07_DuplicateAddress.py

```diff
@@ -117,7 +117,13 @@ def test(self):
         self.nodes[ED2].add_ipaddr('2001::1')
         time.sleep(3)
 
-        self.nodes[ED3].ping('2001::1')
+
+        # Final ping doesn't need a response;
+        # used to trigger an ADDR_ERR.ntf
+        try:
+            self.nodes[ED3].ping('2001::1')
+        except pexpect.TIMEOUT:
```

Comment:
@darconeous, option (3) has been implemented, and now the prefix add call returns success.  The ping still fails with NoRoute however.  Do the `ipaddr add` commands need to set PROP_THREAD_ALLOW_LOCAL_NET_DATA_CHANGE=1 as well, or something similar?